### DISCUSSION TITLE
Newbie Here...Greetings From Niagara Falls...Need A Little Installation Help...Please

### DISCUSSION THREAD

**Discussion by UpperTemagami** [microsoft/vscode-discussions#883] [Extension Development QnA]
Hi,

Thanks for reading.

(1) I’m in a Windows 11 Operating Environment…I initially installed:

Python 3.12 (64-bit)

from the Microsoft Store…

From there…I was able to open Python for the first time with full black screenage…and do a print “Hello Word” test successfully…

(2) I then installed VS Code as my editor…and after that…the Python extension.

(3) I then selected the Interpreter that was recommended…(it matched the version of Python which I had installed)

At this point…I’m at the Terminal…

My “Getting Started Instructions” from Microsoft…tell me to enter the command:

python

But instead of it actually launching python…it hyperlinks me to the Microsoft Store URL that is the general download page for Python.

Any suggestions?

Thanks for your help…it is much appreciated.

-Mike
here's a screenshot from within my terminal:
<img width="693" alt="Screenshot from VS Code" src="https://github.com/microsoft/vscode-discussions/assets/148780406/9730f89a-5627-48d3-9c39-e68662126db8">

**Comment by isidorn** [comment]
This discussion repo is for VS Code extension authors. For all other questions I suggest to go to StackOverflow or ask Bing Chat

**Comment by UpperTemagami** [comment]
more trial and error...got it to work...thanks
